### PR TITLE
Chore: Upgrade Truffle to v5

### LINKED ISSUE #82: Upgrade Truffle to v5
Tackle this task before migrating the contracts codebase to solc 0.5 (#80) and before adding more tests (#81, #83, #84, #85, and https://github.com/aragon/aragon-court/issues/94)

### PR DESCRIPTION
**Pull request by facuspagnuolo**
**MERGE AFTER #103**

Fixes #82

### CODE REVIEW

File: test/court/court-disputes.js [hunk 1]
```diff
@@ -33,14 +33,13 @@ contract('Court', ([_, sender]) => {
   describe('createDispute', () => {
     context('when the given input is valid', () => {
       const draftTermId = 2
-      const jurorsNumber = 10
+      const jurorsNumber = 6
```
[the remark this answers is not in the document]
Comment (facuspagnuolo) [reply test/court/court-disputes.js]: simply to make the tests run faster 😄

File: test/helpers/assertThrow.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  assert(0, `Expected "${expectedErrorCode}"${expectedReason ? ` (with reason: "${expectedReason}")` : ''} but it did not fail`)
+}
+
+async function assertRevert(blockOrPromise, reason) {
+  const error = await assertThrows(blockOrPromise, REVERT_CODE, reason)
+  const errorPrefix = `${THROW_ERROR_PREFIX} ${REVERT_CODE}`
+
+  if (error.message.includes(errorPrefix)) {
+    error.reason = error.message.replace(errorPrefix, '')
+    // Truffle 5 sometimes add an extra ' -- Reason given: reason.' to the error message 🤷
```
Comment (izqui) [inline test/helpers/assertThrow.js]: LOL

File: test/helpers/court.js [hunk 3]
```diff
@@ -294,36 +295,8 @@ module.exports = (web3, artifacts) => {
       await this.passTerms(this.appealConfirmTerms)
     }
 
-    async settle({ disputeId, jurors }) {
```
Comment (izqui) [inline test/helpers/court.js]: Why are we removing it here? Is it no longer used?

Comment (facuspagnuolo) [reply test/helpers/court.js]: Yup

**Pull request merged**